### DISCUSSION TITLE
Tweepy V2 tweet id attribute for tweet list object

### DISCUSSION THREAD

**Discussion by zachthorppp** [tweepy/tweepy#1870] [Questions] [answered]
Hey

I cant find anything on the documentation for the new twitter api v2. 

to get the mention ids for on the old version it was as simple as:

mentions = api.mentions_timeline()
print("Getting mention ids with text") 
for mention in mentions:
    print(str(mention.id) + " - " + mention.text)

is there a simple way to iterate through the mentions list as per below to get the tweet or user_ids:

my_mentions = client.get_users_mentions(id='xxxxxxxxx')

for mention in my_mentions:
    print('str(mention.id))

'list' object has no attribute 'tweet_id'

hope this makes sense!

**Comment by Harmon758** [comment, marked as answer]
[`Client.get_users_mentions`](https://tweepy.readthedocs.io/en/v4.8.0/client.html#tweepy.Client.get_users_mentions) returns a [`Response`](https://tweepy.readthedocs.io/en/v4.8.0/client.html#response) object by default.

When you iterate over it, you're iterating over its fields.

Although, I'm not sure how you'd be getting that error with that code, since you're not attempting to access a `tweet_id` attribute anywhere, and since `print('str(mention.id))` will return a `SyntaxError` with no closing quotation mark.

Regardless, you probably want to iterate over the `data` field of the `Response` instead.

**Reply by zachthorppp** [reply]
Makes perfect sense, Thankyou!
